### PR TITLE
return largest of 3 num

### PR DESCRIPTION
**Pull request by kevinYCKim33**

### CODE REVIEW

File: curriculum/js1/2.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  return Math.max(...[a,b,c]);
```
Comment (seemcat) [inline curriculum/js1/2.js]: Nice try! Solve this problem without the max function.

**Review by seemcat — CHANGES_REQUESTED** [review]

File: curriculum/js1/2.js [hunk 2]
```diff
@@ -7,8 +7,14 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
-};
+  if (b > a && b > c) {
+    return b;
+  } else if (a > b && a > c) {
```
Comment (songz) [inline curriculum/js1/2.js]: don't use else if you have returned
```
if (b > a && b > c) {
  return b;
}
if (a > b && a > c ) {
  return a;
}
return c;
```

Comment (seemcat) [reply curriculum/js1/2.js]: also, please use recursion.

**Review by seemcat — CHANGES_REQUESTED** [review]
turn into a recursive function

**Review by songz — APPROVED** [review]

**Pull request merged**